### PR TITLE
HTBHF-1735 Updated manifest and instructions to include the session details app

### PR DESCRIPTION
**Pull request by YetAnotherMatt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,23 +21,13 @@ THRESHOLD_MEAN_MILLIS - the maximum allowed average response time, in ms
...
 the jar will not stop once the tests are complete, but will wait to be stopped externally (this is managed by scripts in htbhf-continuous-delivery).
 
+See the [`run_performance_tests`](https://github.com/DepartmentOfHealth-htbhf/htbhf-continous-delivery/blob/master/cd_scripts/cd_functions.sh#L171)
+function in htbhf-continuous-delivery to see how the performance tests are deployed and run in the CD pipeline.
+
 Note also that the session details app must be running for performance tests to succeed (this provides access to the 2fa confirmation code).
 See [src/test/session-details-provider/README.md](https://github.com/DepartmentOfHealth-htbhf/htbhf-applicant-web-ui/blob/master/src/test/session-details-provider/README.md)
 in htbhf-applicant-web-ui for more details.
 
-## Running the tests from the command line
```
Comment (YetAnotherMatt) [inline README.md]: This is no longer the recommended way to run the tests

File: manifest.yml [hunk 2]
```diff
@@ -14,4 +14,5 @@ applications:
     PERF_TEST_END_NUMBER_OF_USERS: 50
     PERF_TEST_SOAK_TEST_DURATION_MINUTES: 1
     THRESHOLD_95TH_PERCENTILE_MILLIS: 1000
-    THRESHOLD_MEAN_MILLIS: 500
\ No newline at end of file
+    THRESHOLD_MEAN_MILLIS: 500
+    SESSION_DETAILS_BASE_URL: ((session_details_url))
```
Comment (YetAnotherMatt) [inline manifest.yml]: This will only have an effect when running `cf push` from your local machine, as this manifest is not used by the CD pipeline (which writes its own).

File: README.md [hunk 3]
```diff
@@ -21,23 +21,13 @@ THRESHOLD_MEAN_MILLIS - the maximum allowed average response time, in ms
...
- * RESULTS_DIRECTORY - the directory (relative path) to store the results to.
- * BASE_URL - the url of the UI application to run the tests against.
- * SESSION_DETAILS_BASE_URL - the url of the session details app (this will need to be started before you run the performance tests)
- * PERF_TEST_START_NUMBER_OF_USERS - the number of new users per second at the start of the test
- * PERF_TEST_END_NUMBER_OF_USERS - the number of new users per second at the end of the test
- * PERF_TEST_SOAK_TEST_DURATION_MINUTES - (default 2) - the number of minutes to maintain that level of load, after ramping up
- 
+## Running the tests against your local machine
+First start the session-details-app locally (see above).
 Then run `./run_performance_tests.sh`
```
Comment (mikeSmith08) [inline README.md]: Surely this should be `./run_performance_test_locally.sh`.?

File: run_performance_tests_against_local.sh [hunk 4]
```diff
@@ -15,7 +15,7 @@ mv src/main/scala/uk/gov/dhsc/htbhf/RunGatlingTests.scala src/main/scala/uk/gov/
 
 export BIN_DIR="./bin"
 export GATLING_FOLDER_NAME="gatling-charts-highcharts-bundle-3.0.1.1"
-export GATLING_URL="http://localhost:8080"
+export GATLING_URL="https://repo1.maven.org/maven2/io/gatling/highcharts/gatling-charts-highcharts-bundle/3.0.1.1/gatling-charts-highcharts-bundle-3.0.1.1-bundle.zip"
```
Comment (mikeSmith08) [inline run_performance_tests_against_local.sh]: Why is this change needed?

Comment (YetAnotherMatt) [reply run_performance_tests_against_local.sh]: The GATLING_URL variable is where gatling should be downloaded from. I can't see how this ever worked.

File: README.md (hunk 3 above)
Comment (YetAnotherMatt) [reply README.md]: Changed, thanks.
